### DISCUSSION TITLE
How to install on Debian

### DISCUSSION THREAD

**Discussion by rwb196884** [Netatalk/netatalk#742] [Q&A] [answered]
Give us a clue?

There isn't a package for Debian -- even for 3 let alone 2.

```
# git clone https://github.com/Netatalk/netatalk.git
# cd netatalk
# git pull
# git checkout -b branch-netatalk-2-3
# git pull
# ./bootstrap

**Error**: You must have `libtool' installed.
Get ftp://ftp.gnu.org/pub/gnu/libtool-1.2d.tar.gz
(or a newer version if it is available)
root@calleva:~/netatalk# apt install libtool
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
libtool is already the newest version (2.4.7-5).
libtool set to manually installed.
0 upgraded, 0 newly installed, 0 to remove and 20 not upgraded.
```

**Comment by rdmark** [comment, marked as answer]
Debian is weird in that you need to install a package called `libtool-bin` to get the libtool app.

This article is a good starting point: https://netatalk.io/docs/Installing-Netatalk-2-on-Debian

We recently published a Docker image which is an option if you don't want to bother with building from source:

https://hub.docker.com/r/netatalk/netatalk2

And a final option is to install the deb (amd64 only right now) that's attached to the stable release:

https://github.com/Netatalk/netatalk/releases/tag/netatalk-2-3-1
